### ISSUE TITLE
TIC - Validation of the Addition of Gateway East for Routing of VA.gov Traffics

### ISSUE THREAD

**Issue by raywangoctova** [department-of-veterans-affairs/va.gov-team#68192]
## Description

In talking with NEO, they removed traffic from Gateway West (GWW) to Gateway North (GWN) about a week ago to prepare for the GWW move. Now that all traffic is running explicitly over GWN, 7 a.m. EST on Oct 27th, a physical shutdown will occur to take GWW offline. NEO is going to add Gateway East (GWE) into the mix and serve from both GWN and GWE. The ultimate goal here is to be able to run traffic over both GWN and GWE. 

They have asked for the VA.gov Platform to test some of the more prominent services on 10/24/2023 from 5 PM to 6 PM ET to ensure there are no obvious issues, etc. They are hoping to update the DNS record prior to the completion of the test tomorrow.

## Resources 
Kyle M did the initial test. He should have more information to provide. Clint L has followed this effort from the start and confirmed that DevOps Oncall can support the testing.

## Acceptance Criteria
- [x] VA.gov and Platform services are working as expected.

## Refinement Guidance - Check the following before working on this issue: 
- [ ] _Team label assigned ("platform-tech-team-2")_
- [ ] _Epic assigned (if needed)_ 
- [ ] _Estimated (points assigned)_
- [ ] _Sprint assigned (once planned)_
- [ ] _Team member(s) assigned_
- [ ] _Understands how this work aligns with the overall platform_

**Comment by little-oddball** [comment]
Performed testing and things all look to be flowing as we would expect with the test scenarios we worked through.

Contact Info for a problem:

NOC Network Edge Operations (NEO)
24/7 Phone: 304.262.5282 or 304.262.7785  (both work)
Email: nocnetworkedgeoperations@va.gov
ServiceNow Group: IO.NETWORK.NOC.NEO

We worked w/ Chris Corliss. If a problem arises, use the # above and ask for GWE to be turned off.
